python/cog/json.py

```diff
@@ -24,6 +17,13 @@ def make_encodeable(obj: Any) -> Any:
 
     Somewhat based on FastAPI's jsonable_encoder().
     """
+    try:
+        import numpy as np  # type: ignore
```

Comment:
How do Python imports work? Would subsequent calls to `make_encodable` have the same overhead? Or should we move `has_numpy` to top-level scope and conditionalize the import?

Reply:
Also, would it make sense to move this to the bottom of the conditional, like replaying `if has_numpy` with an `else`?